src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -378,5 +378,8 @@
     <Reference Include="System" />
     <Reference Include="System.Drawing" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="System\Drawing\Internal\GpPathData.cs" />
```

Comment:
If this is a Windows specific could you move it up to the ItemGroup conditioned to TargetWindows L207? if we plan to use this in Unix code as well, could you please move it up the the common ItemGroup L29?

Reply:
Sure